solutions/devsprint-eduardo-bocato-1/SpaceApp/Modules/Features/Home/Scenes/Home/HomeScene.swift

```diff
@@ -9,14 +9,33 @@ struct HomeScene: View {
     }
     
     var body: some View {
-        NavigationView {
-            List(viewModel.launches) { launch in
-                Text("\(launch.name)")
+        TabView {
```

Comment:
- How do we know which tab is selected? 
Let's imagine I want to send an Analytics event when the tab changes... Is this ready for it?

- What  if I want to programatically select a tab? Is how can we do that?

Reply:
Oh, sorry, I forgot to bind an indicator to know which tab is selected and for analytics we can use `onChange` to execute code when a state variable changes. I'll update with the suggestions, thank you :)